packages/libsail/libsail.0.15/opam

```diff
@@ -0,0 +1,68 @@
+opam-version: "2.0"
+synopsis:
+  "Sail is a language for describing the instruction semantics of processors"
+description: """
+Sail is a language for describing the instruction-set
+architecture (ISA) semantics of processors. Sail aims to provide a
+engineer-friendly, vendor-pseudocode-like language for describing
+instruction semantics. It is essentially a first-order imperative
+language, but with lightweight dependent typing for numeric types and
+bitvector lengths, which are automatically checked using Z3. It has
+been used for several papers, available from
+http://www.cl.cam.ac.uk/~pes20/sail/.
+"""
+maintainer: ["Sail Devs <cl-sail-dev@lists.cam.ac.uk>"]
+authors: [
+  "Alasdair Armstrong"
+  "Thomas Bauereiss"
+  "Brian Campbell"
+  "Shaked Flur"
+  "Jonathan French"
+  "Kathy Gray"
+  "Robert Norton"
+  "Christopher Pulte"
+  "Peter Sewell"
+  "Mark Wassell"
+]
+license: "BSD-2-Clause"
+homepage: "https://github.com/rems-project/sail"
+bug-reports: "https://github.com/rems-project/sail/issues"
+depends: [
+  "dune" {>= "3.0"}
```

Comment:
```suggestion
  "ocaml" {>= "4.08" & < "5.0"}
  "dune" {>= "3.0"}
```

This suggestion is based on the result of the CI. I think it applies to the other packages as well.

Reply:
Instead of the 5.0 constraint, it might be sufficient to fix dep packages such as https://github.com/ocaml/opam-repository/pull/22593

Reply:
I think the ott package may also have the same issue

Reply:
I've sorted out the deps upstream.  We haven't released them yet, but when we do this version of Sail should compile with 5.0 (if my local copy is anything to go by).